server/views/temporary-accommodation/dashboard/index.njk

```diff
@@ -20,6 +20,15 @@
       </div>
     </div>
 
+    <div class="card">
+      <div class="card-body">
+        <h2 class="govuk-heading-s">
+            <a class="govuk-link" href="{{ paths.bookings.search({}) }}">Find a booking</a>
+        </h2>
+        <p class="govuk-body">View and manage provisional, confirmed, active and closed bookings.</p>
+        </div>
+    </div>
+
```

Comment:
Should have thought about this with my PR, but think it's worth moving "download" data to be last as part of this